### PR TITLE
Test and fix Travis CI build.

### PR DESCRIPTION
**Pull request by schlessera**

### CODE REVIEW

File: ci/test.sh [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-
 # Run the functional tests for the framework
+BEHAT_TAGS=$(BEHAT_FEATURES_FOLDER=vendor/wp-cli/wp-cli/features php ci/behat-tags.php)
 vendor/bin/behat --format progress $BEHAT_TAGS --strict
 
 # Run the functional tests for the bundled commands
-REPOS=$(cat composer.json | grep -m 1 -oE "wp-cli/([a-z\-]*)-command")
+REPOS=$(cat vendor/wp-cli/wp-cli/composer.json | grep -oE "wp-cli/([a-z\-]*)-command")
 for REPO in $REPOS; do
+	BEHAT_TAGS=$(BEHAT_FEATURES_FOLDER=vendor/$REPO/features php ci/behat-tags.php)
```
Comment (danielbachhuber) [inline ci/test.sh]: If you have `behat.yml,` why do you need to do this?

Comment (schlessera) [inline ci/test.sh]: I don't understand what you mean. The behat tags script does not parse the `behat.yml` file.

**Comment by danielbachhuber** [comment]
I canceled the build because it's going to take forever to run.

Can we only run this variation?

image

File: ci/test.sh (hunk 1 above)
Comment (danielbachhuber) [inline ci/test.sh]: Ah, nevermind. I understand what's going on now.

**Comment by schlessera** [comment]
It would have just stopped at the first step, because there's still an issue with the Phar paths.

But yes, I can reduce the matrix.

**Comment by schlessera** [comment]
So, the entire framework now runs through the tests successfully. There's still an issue with the external commands not having the autoloader in Travis, looking into that one now.

**Pull request merged**